WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/CategorySectionTableViewCell.swift

```diff
@@ -22,14 +22,15 @@ protocol CategorySection {
     var description: String? { get }
     var thumbnails: [Thumbnail] { get }
     var scrollOffset: CGPoint { get set }
+    var thumbnailSize: CGSize { get }
 }
 
 class CategorySectionTableViewCell: UITableViewCell {
 
     static let cellReuseIdentifier = "\(CategorySectionTableViewCell.self)"
     static let nib = UINib(nibName: "\(CategorySectionTableViewCell.self)", bundle: Bundle.main)
-    static let expectedThumbnailSize = CGSize(width: 160.0, height: 240)
-    static let estimatedCellHeight: CGFloat = 310.0
+    static let defaultThumbnailSize = CGSize(width: 160, height: 240)
+    static let cellVerticalPadding: CGFloat = 100
```

Comment:
The cell height can now vary depending on the thumbnail size, so this now only provides the vertical padding that would be added to the cell.